rj_gameplay/rj_gameplay/action/kick.py

```diff
@@ -16,19 +19,22 @@ def done(self) -> bool:
 class Kick(IKick):
     """
     Kick action
-    TODO: update with actions implementation
     """
-    def __init__(self, point: np.ndarray):
-        self.point = point
-        self.count = -1
-        #for stub
-
-    def tick(self, robot: rc.Robot, ctx: action.Ctx) -> None:
-        print('robot:', robot.id, 'kicking')
-        self.count += 1
-
-    def done(self) -> bool:
-        return self.count == 1
-
-    def fail(self):
-        return False
+    def __init__(self, robot_id:Optional[int]=None, chip:Optional[bool]=False, kick_speed:Optional[float]=255.0) -> None:
```

Comment:
You would want to lower the kick_speed here?

Reply:
`kick_speed` should be in m/s, so this is definitely way too fast. Do these all really need default values? I'd prefer not to allow a default value for kick at least, and I thought we designed things in a way that you didn't need to construct actions until you know which robot they're assigned to?